include/stl2/view/drop_while.hpp

```diff
@@ -65,7 +78,7 @@ STL2_OPEN_NAMESPACE {
 			R base_;
 		};
 
-		template <Range R, IndirectPredicate<iterator_t<R>> Pred>
+		template <class R, ext::CopyConstructibleObject Pred>
```

Comment:
`ext::CopyConstructibleObject` shouldn't be necessary here: the `drop_while_view` specialization applies that constraint for us. (Well, sort of. It requires `CopyConstructible && std::is_object_v` which is equal to `ext::CopyConstructible` but not equivalent to subsumption in post-[P0717](https://wg21.link/P0717) world. Since subsumption isn't involved here anywhere, the difference is moot.)